tslint.json

```diff
@@ -0,0 +1,39 @@
+{
+  "extends": ["tslint:recommended", "tslint-react", "tslint-config-prettier", "tslint-immutable"],
+  "rules": {
+    "interface-name": false,
+    "member-ordering": false,
```

Comment:
I think it would be great to get this to true too, however I don't know if this would have downsides?

Reply:
I would not do it; for me it is important to be able to semantically group props together, from "most important" to "less important"

Reply:
@skurfuerst 
* `member-ordering` refers to `class` members: [docs](https://palantir.github.io/tslint/rules/member-ordering/)
* `object-literal-sort-keys` refers to order in `objects`: [docs](https://palantir.github.io/tslint/rules/object-literal-sort-keys/)
  * I think this is what you mean
* `ordered-imports` refers to order of `imports`: [docs](https://palantir.github.io/tslint/rules/ordered-imports/)

Right now _all_ of them are set to `true`.
Any suggestions, preferences?